apps/account-api/test/accounts.v2.controller.e2e-spec.ts

```diff
@@ -52,114 +53,288 @@ describe('Accounts v2 Controller', () => {
+                  type: 'JsonSchema',
+                  id: 'https://schemas.frequencyaccess.com/VerifiedEmailAddressCredential/bciqe4qoczhftici4dzfvfbel7fo4h4sr5grco3oovwyk6y4ynf44tsi.json',
+                },
+                credentialSubject: {
+                  id: 'did:key:z6QNucQV4AF1XMQV4kngbmnBHwYa6mVswPEGrkFrUayhttT1',
+                  emailAddress: 'john.doe@example.com',
+                  lastVerified: '2024-08-21T21:27:59.309+0000',
+                },
+                proof: {
+                  type: 'DataIntegrityProof',
+                  verificationMethod: 'did:web:frequencyaccess.com#z6MkofWExWkUvTZeXb9TmLta5mBT6Qtj58es5Fqg1L5BCWQD',
+                  cryptosuite: 'eddsa-rdfc-2022',
+                  proofPurpose: 'assertionMethod',
+                  proofValue:
+                    'z4jArnPwuwYxLnbBirLanpkcyBpmQwmyn5f3PdTYnxhpy48qpgvHHav6warjizjvtLMg6j3FK3BqbR2nuyT2UTSWC',
+                },
+              },
+              {
+                '@context': [
+                  'https://www.w3.org/ns/credentials/v2',
+                  'https://www.w3.org/ns/credentials/undefined-terms/v2',
+                ],
+                type: ['VerifiedGraphKeyCredential', 'VerifiableCredential'],
+                issuer: 'did:key:z6QNucQV4AF1XMQV4kngbmnBHwYa6mVswPEGrkFrUayhttT1',
+                validFrom: '2024-08-21T21:28:08.289+0000',
+                credentialSchema: {
+                  type: 'JsonSchema',
+                  id: 'https://schemas.frequencyaccess.com/VerifiedGraphKeyCredential/bciqmdvmxd54zve5kifycgsdtoahs5ecf4hal2ts3eexkgocyc5oca2y.json',
+                },
+                credentialSubject: {
+                  id: 'did:key:z6QNucQV4AF1XMQV4kngbmnBHwYa6mVswPEGrkFrUayhttT1',
+                  encodedPublicKeyValue: '0xb5032900293f1c9e5822fd9c120b253cb4a4dfe94c214e688e01f32db9eedf17',
+                  encodedPrivateKeyValue: '0xd0910c853563723253c4ed105c08614fc8aaaf1b0871375520d72251496e8d87',
+                  encoding: 'base16',
+                  format: 'bare',
+                  type: 'X25519',
+                  keyType: 'dsnp.public-key-key-agreement',
+                },
+                proof: {
+                  type: 'DataIntegrityProof',
+                  verificationMethod: 'did:key:z6MktZ15TNtrJCW2gDLFjtjmxEdhCadNCaDizWABYfneMqhA',
+                  cryptosuite: 'eddsa-rdfc-2022',
+                  proofPurpose: 'assertionMethod',
+                  proofValue:
+                    'z2HHWwtWggZfvGqNUk4S5AAbDGqZRFXjpMYAsXXmEksGxTk4DnnkN3upCiL1mhgwHNLkxY3s8YqNyYnmpuvUke7jF',
+                },
+              },
+            ],
+          }),
+        ),
+      };
+
+      const response = await request(httpServer).post('/v2/accounts/siwf').send(mockPayload).expect(HttpStatus.OK);
+
+      expect(response.body).toHaveProperty('controlKey');
+      expect(response.body).toHaveProperty('rawCredentials');
+    });
+
+    // TODO: Is there a way to mock the fetch in the e2e?
```

Comment:
Is there a way to do this? I don't think there is which means we would need to setup a mock server.

Worth doing it? I can do it fairly easily as we already have a mock server system and clean response json

Reply:
If the merge is not urgent then I feel it will be nice to have your piece of mock server already in place, will come handy as we develop more.

Reply:
I'll see what I can do on Monday.

Reply:
Added!